### PR TITLE
COMMERCE-7856, COMMERCE-7686, COMMERCE-7684, COMMERCE-7688, COMMERCE-7689, COMMERCE-7700 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/COMMERCE-7856
https://issues.liferay.com/browse/COMMERCE-7686
https://issues.liferay.com/browse/COMMERCE-7684
https://issues.liferay.com/browse/COMMERCE-7688
https://issues.liferay.com/browse/COMMERCE-7689
https://issues.liferay.com/browse/COMMERCE-7700

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 1]
```diff
@@ -133,14 +133,74 @@ definition {
 		}
 	}
 
-	@description = "[Please Update]"
-	@ignore = "Test Stub"
+	@description = "Create multiple orders and use the search bar to verify if it works correctly"
```
Comment (Kvale1733) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:541]: This should be something like "Verify orders can be search when importing an order"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 2]
```diff
@@ -133,14 +133,74 @@ definition {
...
 	@priority = "4"
```
Comment (Kvale1733) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:542]: If priority isn't specified in the ticket you should set it to 3

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 3]
```diff
@@ -133,14 +133,74 @@ definition {
...
+					productName = "${productName}",
+					productQuantity = "1",
+					productUnitPrice = "${productUnitPrice}",
+					shippingMethod = "Standard");
+
+				CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+			}
+		}
+
+		task ("When the buyer tries to import an order") {
```
Comment (Kvale1733) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:585]: This should also be a given task. "And given the user is importing an order"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 4]
```diff
@@ -133,14 +133,74 @@ definition {
...
+			var orderIds = CommerceJSONOrdersAPI._getCommerceOrderIds(orderCount = "3");
 
+			for (var orderId : list "${orderIds}") {
+				Search.searchCP(searchTerm = "${orderId}");
+
+				AssertElementPresent(
+					key_entryName = "${orderId}",
+					locator1 = "CommerceEntry#TABLE_LIST_TITLE");
+			}
+		}
```
Comment (Kvale1733) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:607]: This should be two separate tasks.

1. When the buyer searches the ID of a completed order
2. Then only that order should appear

Also. Can you add asserts that the other orders aren't present. You only need to search for one order.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 5]
```diff
@@ -203,13 +203,43 @@ definition {
...
+		task ("Given a Minium site with a buyer user") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And the user go to create a new order") {
```
Comment (Kvale1733) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:624]: This should be "And given the user is creating a new order"

**Comment by Kvale1733** [comment]
Also Afonso, the commits are a bit mixed up. Seems a lot of the tests is part of the SF. An SF commit should be for simple formatting fixes.

Also, for the commit messages, can you add the test ticket to each commit. They shouldn't all be the same ticket.

**Comment by AfonsoHSantos** [comment]
Hi @Kvale1733, thanks for the suggestions! I made the changes you asked for!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
Rebased and reopened here for testing: https://github.com/Kvale1733/liferay-portal/pull/90

**Comment by Kvale1733** [comment]
ci:close

**Pull request closed without merging**